test-requirements.txt

```diff
@@ -35,3 +35,4 @@ pyrabbit
 pip-tools<5.4
 pytest==5.4.3
 pytest-benchmark==3.2.3
+zstandard==0.15.1
```

Comment:
Where did this come from? I'd add a comment noting where it's used, or what depends on it.

Reply:
It's test required for micro benchmarks, will add a test.